client/src/__tests__/RequestBouquet.test.jsx

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from "@testing-library/react";
+import { BrowserRouter as Router } from 'react-router-dom';
+// import userEvent from '@testing-library/user-event';
+
+
+import RequestBouquet from "../components/RequestBouquet";
+
+test ("buttons render", () => {
+    render(
+    <Router>
+        <RequestBouquet />
```

Comment:
This is more of a general comment. Do you need to pass down props to this component to get it to work? Think about it!